packages/components/molecules/f-searchbox/src/components/Form.vue

```diff
@@ -30,7 +30,8 @@
                 :is-compressed="isCompressed" />
         </div>
 
-        <form-search-suggestions
+        <component
+            :is="setSuggestionType()"
```

Comment:
Do you need the parentheses here?

Reply:
It was a method but probably should have been a computed prop so updated it now 👍

Reply:
Ah yeah, that makes sense 👍🏻